internal/character/clara/ult.go

```diff
@@ -0,0 +1,73 @@
+package clara
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Ult        key.Modifier = "clara-ult" // MAvatar_Klara_00_Ultra_WarriorMode
+	UltBuff    key.Modifier = "clara-ult-buff"
+	UltCounter key.Modifier = "clara-ult-enhanced-counter"
+)
+
+func init() {
+	modifier.Register(Ult, modifier.Config{
+		BehaviorFlags: []model.BehaviorFlag{model.BehaviorFlag_STAT_BURST},
+		Stacking:      modifier.Refresh,
+		Duration:      2,
+	})
+
+	modifier.Register(UltBuff, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+		Duration:   2,
+	})
```

Comment:
Merge these into a single buff. No reason to separate if their signature is the same (they are also a single modifier in the datamine)